cockroachcloud/cloud-api.md

```diff
@@ -441,7 +441,7 @@ Where:
 
 ## List all clusters in an organization
 
-To list all active clusters within an organization, send a `GET` request to the `/v1/clusters` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](console-access-management.html#service-accounts) to list an organization's clusters.
+To list all active clusters within an organization, send a `GET` request to the `/v1/clusters` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](managing-access.html#manage-service-accounts) to list an organization's clusters.
```

Comment:
In the new auth model, this can be done by Cluster Admin role at the organization scope, or through the Org Admin (legacy) role.